torch_xla/distributed/xla_dist.py

```diff
@@ -67,6 +68,15 @@ def __init__(self,
                conda_env=None,
                env_vars=None):
     self._cluster = cluster
+    client_master_ip = ClusterResolver._get_instance_metadata(
+      'instance/network-interfaces/0/ip')
+    self._client_master = next(filter(
+      lambda cw: cw._internal_ip == client_master_ip,
+      self._cluster._client_workers))
```

Comment:
Another private member access (_cluster is fine, _client_workers is not).

Reply:
Made the member public since we access it from quite a few places.